### PR TITLE
Task 6

### PR DESCRIPTION
**Pull request by HashimElAlej**
fixed previous issues

### CODE REVIEW

**Review by nickkagame — CHANGES_REQUESTED** [review]
Hi Hashim, Some good work here.  Though we still need a bit more.  I am not seeing any testing for errors.  Your code will also need to check if an article that you want comments for actually exists so you give the correct response (e.g. 404 for an article that doesn't exist, and 200 for an article and responding with empty array for an article that does exist but has no comments)

File: models/app.model.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            }
+            return result
+        });
+}
+
+exports.findAllArticles = () => {
+    return db.query(`
+    SELECT articles.article_id,articles.title,articles.topic,articles.author,articles.body,articles.created_at,articles.votes,articles.article_img_url, COUNT(comments.article_id) AS comment_count
+    FROM articles
+    JOIN comments ON articles.article_id = comments.article_id
```
Comment (Keb84) [inline models/app.model.js:39]: You're going to need left join here. If you don't it will mean any article without comments won't be included

**Review by Keb84 — APPROVED** [review]
Nice work 🚀 I've just made one quick comment. Please have a look and merge when ready 🙂

**Review by johnhutcheon — CHANGES_REQUESTED** [review]
Thanks for making these changes Hashim. There's still one more test that Nick specified that you haven't added though- **200 for an article and responding with empty array for an article that does exist but has no comments**
Could you please add and resubmit? 👍
